File: algoritms/fibonancy/basic-recursion.rb
def time_diff_milli(start, finish)
   (finish - start) * 1000.0
end

def fibo(n)
  if (n == 0)
    return 0
  elsif (n == 1)
    return 1
  else
    return fibo(n-1) + fibo(n-2)
  end
end

t1 = Time.now
40.times do |i|
  ret = fibo(i)
  puts "F" + i.to_s + " : " + ret.to_s
end

t2 = Time.now

msecs = time_diff_milli t1, t2

puts ""
puts "Start Time: " + t1.to_s
puts "End Time: " + t2.to_s
puts "Execute Time: " + msecs.to_s + "ms"

File: algoritms/fibonancy/basic-recursion.js
function fibo(n) {
  if (n === 0){
    return 0;
  }else if (n === 1) {
    return 1;
  }else{
    return fibo(n-1) + fibo(n-2);
  }
}

console.time("watcher");
for (var i = 0; i < 40; i++) {
  console.log("F" + i + " : " + fibo(i));
}
console.timeEnd("watcher");
